src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocNodeImpl.java

```diff
@@ -72,6 +72,20 @@
      */
     private DetailNode parent;
 
+    /**
+     * Reflects if any of the successors of this {@code DetailNode} has property
+     * {@link #unclosedTag} set true.
+     */
+    private Boolean hasUnclosedTags;
```

Comment:
fair enough. how about calling the field `hasTagUnclosed` or `containsUnclosedTag`